Packages/MIES/MIES_Utilities.ipf

```diff
@@ -3144,10 +3144,12 @@ End
 ///
 /// @param wv     numeric wave
 /// @param sep    separator
+/// @param colSep [optional, default = `,`] separator for column entries
 /// @param format [optional, defaults to `%g`] sprintf conversion specifier
-threadsafe Function/S NumericWaveToList(WAVE/Z wv, string sep, [string format])
-
+threadsafe Function/S NumericWaveToList(WAVE/Z wv, string sep, [string format, string colSep])
```

Comment:
@t-b are in-line return parameters (e.g., ```Function/S [string list] NumericWaveToList(WAVE/Z wv, string sep, [string format, string colSep])``` ) only used when there is >1 return param?

Reply:
Jeep. This is done as using the single-return signature is nicer at the call site.

In the following example three different use cases are tried with classic and MRS (multiple return value syntax) functions. Only the classic one can be used in all three cases, MRS fails in two.

```igorpro
Function/S FuncClassic()

	return ""
End

Function [string str] FuncMRS()

	return [""]
End

Function Consume(string str)

End

Function Test()
	string str1, str2

	// ignoring return value works
	FuncClassic()

	// using it
	str1 = FuncClassic()

	// pass onto caller
	Consume(FuncClassic())

	// does not compile
	// FuncMRS()
	
	[str2] = FuncMRS()
	
	// does not compile
	// Consume(FuncMRS())
End

```